conf/nuodb.conf

```diff
@@ -3,15 +3,15 @@
 #  the helm chart.  probably want to put into different files.
 
 [[inputs.execd]]
-  command = [ "python", "/opt/nuocd/nuodb_adminquery.py", "-i", "0", "metrics" ]
+  command = [ "$PYTHON2", "/opt/nuocd/nuodb_adminquery.py", "-i", "0", "metrics" ]
```

Comment:
this change is not forward compatible with our Kubernetes helm charts. Kubernetes uses nuodb-collector:latest and this requires a change to the charts to expose a new environmental variable. If we released nuodb-collector, it would break the Kubernetes experience.